packages/website/src/pages/ContentPage/DocPage.tsx

```diff
@@ -35,6 +35,19 @@ function isModifiedEvent(event) {
 
 const monoFF = ['source code', 'courier', 'mono'];
 
+function replaceUnsupportedFonts(el: HTMLElement) {
+  const ff = el.style.fontFamily;
+  const fs = el.style.fontSize;
+  if (ff && fs) {
+    const fonts = (document as any).fonts;
+    var font = `${fs} ${ff}`;
+    if (fonts && fonts.check && !fonts.check(font)) {
+      // console.log('replacing unsupported font', font, el, el.textContent);
+      el.style.fontFamily = `${fs} Helvetica`;
```

Comment:
How about using the same font as the Wiki itself? It's wired that the UI has font A while the content has font B.

Reply:
Okay. I think if we delete the font family it will inherit the outer font which eventually goes back to that of the wiki.